src/api/redirect-gather-sy-office.ts

```diff
@@ -0,0 +1,13 @@
+import { GatsbyFunctionRequest, GatsbyFunctionResponse } from 'gatsby';
+
+const GATHER_URL = `https://gather.town/app/ea0xvXaHYWuWurME/satellytes`;
+
+/*
+Provide an endpoint to redirect to our gather space. 
+ */
+export default async function handler(
```

Comment:
that works:
https://satellytescommain21751-gatherredirect.gtsb.io/api/redirect-gather-sy-office